src/1-Puzzle/02/assets/stylesheets/object/projects/_index.css

```diff
@@ -0,0 +1,83 @@
+.p-wrap {
+  width: 400px;
+  margin: 0 auto;
+}
+
+.p-header {
+  width: 100%;
+  height: 100px;
+  background-color: #141f40;
+}
+
+.p-index {
+  display: flex;
+}
+
+.p-sidebar {
+  width: 150px;
+  height: 250px;
+  padding: 20px;
+  background-color: #80bfa8;
+}
+
+.p-sidebar_text {
```

Comment:
[must]
BEMの書き方になっていない

```suggestion
.p-sidebar__text {
```

Reply:
修正しました。